### PR TITLE
Add ruledev bundle and app

### PR DESCRIPTION
**Pull request by karelhala**
~~I am not sure about the id of sub app. The url should look like `cloud.redhat.com/ruledev` that's why the ID is empty.~~ Fixed

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -228,6 +228,21 @@ rhel:
...
   top_level: true
 
+ruledev:
+  title: Rule Development
+  channel: '#flip-mode-squad'
+  deployment_repo: https://github.com/RedHatInsights/ruledev-frontend-build
+  frontend:
+    title: Rule Development
+    paths:
+      - /ruledev
```
Comment (kruai) [inline main.yml]: This should be the full path to the app. If we're putting it in the Staging bundle, that's `/staging/ruledev`.

File: main.yml [hunk 2]
```diff
@@ -228,6 +228,21 @@ rhel:
...
+    sub_apps:
```
Comment (kruai) [inline main.yml]: Is this an app group? This field should only exist if you want a sub app that exists under the `ruledev` app.

File: main.yml [hunk 3]
```diff
@@ -228,6 +228,21 @@ rhel:
...
+      - id: 'ruledev'
```
Comment (kruai) [inline main.yml]: If you add a sub app with an ID that points to the original app, we'll end up with an infinite loop when trying to build the navigation.

File: main.yml [hunk 4]
```diff
@@ -228,6 +228,21 @@ rhel:
...
+        title: Home
+        default: true
+        group: ruledev
+  top_level: true
```
Comment (kruai) [inline main.yml]: `top_level` should only be defined if this is a bundle, or something at that level. I'm against that for this app, unless we hear from the higher-ups that `ruledev` needs to be its own separate category.

**Comment by karelhala** [comment]
@kruai not for now at least, we are going to query JIRA directly (at least for now)

**Pull request merged**